antora-playbook-staging.yml

```diff
@@ -183,6 +183,8 @@ asciidoc:
     sqlpp: SQL++
     sqlpp_url: https://www.couchbase.com/products/n1ql
     cbpp: Couchbase++
+    db: cluster
+    cptl-db: Cluster
```

Comment:
As someone who has enjoyed figuring out what attributes like `{sgw-te}` and `{cblFrmWk}` in Mobile mean, I'm excited about everyone else getting the opportunity to participate in the fun!

(Don't mean to snark, I've perpetrated enough of these myself... anyway, seeing it in a PR makes me wonder if we should try to do something about it and maybe come up with some best practices ongoing.)

e.g. What does `cptl` mean? Can we spell it out in full?
Or is it worth adding YAML comments, like e.g.:

`db: cluster   # the name of a "database-like-thing"`

Reply:
Could be listed in the README?

(I'm in the process of doing just that for SDK Docs' own growing list of attributes)

Reply:
I was trying to keep it short for the sake of easy typing in running doc. 

If we'd rather spell it out for explicitness, I'm okay with that.

Reply:
an inline comment would be wonderful from my PoV